devicon.json

```diff
@@ -11278,12 +11278,26 @@
                 "plain-wordmark"
             ]
         },
-        "color": "#f7a41d",
-        "aliases": [
-            {
-                "base": "original",
-                "alias": "plain"
-            }
-        ]
+        "color": "#294172", 
+        "aliases": [] 
+    },
+    {
+        "name": "stackblitz",
+        "tags": [
+            "IDE",
+            "FullStack Development Online",
+            "online ide"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "plain"
```

Comment:
There is no plain icon here. Since the original icon can be made to fit the requirements for plain icons, this can be changed to `original`, once my suggestions for `stackblitz-original.svg` have been implemented